### DISCUSSION TITLE
generate model doesn't show all the column for base model

### DISCUSSION THREAD

**Discussion by paritoshg2010** [AltimateAI/vscode-dbt-power-user#443] [Q&A]
Hello Everyone,

When we click on generate model it doesn't always shows all the column. I believe when the number of columns in source model is huge, we don't get any column in base model. Please refer to the below screenshot.

![image](https://github.com/innoverio/vscode-dbt-power-user/assets/45953849/d659be7f-e8ef-4629-a292-a9f39b81bfca)

Does anyone know if there is any limitation with the use of generate model feature ?

**Comment by lbiedma** [comment]
I am having the same issue, but it happens on some Models and not on others, it's a very strange behavior...

**Reply by anandgupta42** [reply]
Hi @lbiedma and @paritoshg2010 - Can you please create an issue - https://github.com/innoverio/vscode-dbt-power-user/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml ?

Thanks,
Anand

**Reply by lbiedma** [reply]
Hi there, sorry for the late reply, at least in my case it was a question of knowing about the minimal permissions to assign to the user that works with the warehouse, so it got solved.

**Comment by anandgupta42** [comment]
@paritoshg2010 Following up on this, are you still facing this issue?

**Reply by jurgen-hintze** [reply]
This is still an issue, it seems  columns defined as nvarchar are not in the model

**Reply by jurgen-hintze** [reply]
not nvarchar is the problem but user defined datatype

**Reply by anandgupta42** [reply]
@wokkelodio Can you please go ahead and file a bug with the steps to reproduce the problem?

**Reply by wm-NWilliamson** [reply]
I just came across the same error after using the codegen for generate_source and then using generate_model.
This package writes everything in lower case, and this caused it to not find my tables where I had some upper case. After changing the yml file table names to have matching case to the bigquery tables it worked.

Hope the same works for you
